src/extension/content-script/onendnostr.js

```diff
@@ -1,7 +1,6 @@
-import browser from "webextension-polyfill";
-
 import getOriginData from "./originData";
 import shouldInject from "./shouldInject";
+import browser from "webextension-polyfill";
```

Comment:
why did this get all changed? 

didn't we have the config somewhere that first there are extenal package imports and then the internal ones?